lib/src/kvs/tx.rs

```diff
@@ -794,10 +793,15 @@ impl Transaction {
 	/// This function fetches key-value pairs from the underlying datastore in batches of 1000.
 	pub async fn getr<K>(&mut self, rng: Range<K>, limit: u32) -> Result<Vec<(Key, Val)>, Error>
 	where
-		K: Into<Key> + Debug,
+		K: Into<Key> + Debug + Clone,
 	{
 		#[cfg(debug_assertions)]
 		trace!("Getr {:?}..{:?} (limit: {limit})", rng.start, rng.end);
+		println!(
+			"Getr {:?}..{:?} (limit: {limit})",
+			debug::sprint_key(&rng.start.clone().into()),
+			debug::sprint_key(&rng.end.clone().into())
+		);
```

Comment:
I think println! is not going under debug_assertions. It should be grouped in a block
```suggestion
		#[cfg(debug_assertions)]
		{
			trace!("Getr {:?}..{:?} (limit: {limit})", rng.start, rng.end);
			println!(
				"Getr {:?}..{:?} (limit: {limit})",
				debug::sprint_key(&rng.start.clone().into()),
				debug::sprint_key(&rng.end.clone().into())
			);
		}
```